docs/release_process.md

```diff
@@ -40,8 +40,18 @@ Let's make sure there are no migration conflicts with the previous release:
     $ git checkout master
     $ make migrate
```

Comment:
Hum. After we've make the `0.22.0` the data will be there no ? So for the `0.23.0` release (using the `0.22.0` documentation) the `make test-data-migrations` will work.
Or I'm missing something ? (In this case , we should change the next line `Starting with Ideascube 0.22.0, you can instead run:`  who imply that it will work with `0.22.0`)